user-management-react-app/src/reducers/userData.js

```diff
@@ -0,0 +1,28 @@
+const intialData = {
+  isUserLoggedIn: false,
+  userLoginData: {},
+};
+
+const userData = (state = intialData, action) => {
+  const { type, payload } = action;
+  switch (type) {
+    case 'SUBMIT_FORM': {
```

Comment:
no need of {},
you can directly write return statement
case 'SUBMIT_FORM':
  return {
  ...
  };